big_tests/tests/dynamic_modules.erl

```diff
@@ -31,26 +33,25 @@ to_replace([RequiredModule | Rest], CurrentModules, ReplaceAcc, ReplaceWithAcc)
 restore_modules(Domain, Config) ->
     SavedModules = ?config(saved_modules, Config),
     CurrentModules = get_current_modules(Domain),
-    escalus_ejabberd:rpc(gen_mod_deps, replace_modules,
-                         [Domain, CurrentModules, SavedModules]).
+    rpc(mim(), gen_mod_deps, replace_modules, [Domain, CurrentModules, SavedModules]).
 
 get_current_modules(Domain) ->
-    escalus_ejabberd:rpc(gen_mod, loaded_modules_with_opts, [Domain]).
+    rpc(mim(), gen_mod, loaded_modules_with_opts, [Domain]).
 
 stop(Domain, Mod) ->
     Node = escalus_ct:get_config(ejabberd_node),
     stop(Node, Domain, Mod).
 
 stop(Node, Domain, Mod) ->
     Cookie = escalus_ct:get_config(ejabberd_cookie),
-    IsLoaded = escalus_ct:rpc_call(Node, gen_mod, is_loaded, [Domain, Mod], 5000, Cookie),
+    IsLoaded = escalus_rpc:call(Node, gen_mod, is_loaded, [Domain, Mod], 5000, Cookie),
```

Comment:
Looks like 2 ways of using `rpc` is mixed in this module. The one imported from `distributed_helper` and direct use of `escalus_rpc:call`. Are you going to unify it in this PR or is there a reason to do so?